packages/epo-widget-lib/src/widgets/ColorTool/ImageComposite/ImageComposite.tsx

```diff
@@ -37,10 +52,13 @@ const ImageComposite = forwardRef<
       [imagesLoaded, filters]
     );
 
+    const handleFilterChange = (index: number, canvas: HTMLCanvasElement) => {
+      layers.current[index] = canvas;
```

Comment:
puts the current canvas into an array in a ref so state is not affected but the composite renderer can access the layers